modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+
+To use `Scheduler` as an input source:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
++
+For example, the following configuration directs `Scheduler` to:
+
+* Wait 10 seconds before initiating the flow for the first time after the app starts
+* Initiate the flow every 30 seconds after the initial 10 second delay
++
+.Select `Fixed Frequency` in the *Scheduling Strategy* field to initiate a flow at regular intervals.
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to log `my message` every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
+image::introduction/intro-config-fd-scheduler-cron.png[Configuration using a cron expression]
+
+For more information about configuring `Scheduler`, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint]
+
+
+[[connector-specific]]
+=== Use a Connector-Specific Input Source
+
+If a connector has associated input sources, you can use a connector input source to initiate the flow. For example, you can use `Amazon SQS Receive Messages` as an input source for Anypoint connector for Amazon SQS (Amazon SQS Connector). This input source initiates a flow each time the configured Amazon SQS queue receives a new message.
+
+To start the configuration for connector-specific input sources, follow these steps:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Trigger* field, search for and select the connector.
+. If the connector has multiple input sources, select the input source to use. (If the connector has one input source, that input source is selected automatically.)
+
+Each connector-specific input source has a unique configuration, but most configurations require the following actions:
+
+* Completing the required fields.
+* Select an existing connection configuration or adding a new one.
+
+For more information about configuring a connector-specific input source, see the reference guide for the associated connector.
+
+== Configure the Target Connector Operation
+
+After you configure an input source for a connector, you can configure a connector operation as the target component:
+
+. Click *+* next to the trigger card of the input source.
++
+.Clicking the + icon brings up a screen where you can select the target connector and operation.
+image::introduction/intro-config-fd-plus-target.png[Click the Plus Sign]
++
+. In the *Select a component* screen, search for and select the target connector.
+. Select the target operation.
+. On the *General* tab, complete the required fields.
```

Comment:
I believe it pops you immediately into the Connection Configuration card. You either need to select an existing connection or create a new one before you proceed to the general tab.

Reply:
Ashley is right. 
Search for the target connector
Select the operation
You are in the connection configuration, where you can add your new connection, or select an existing connection. 
So, long story short, I think you can just remove the step on line 124.